components/ntp_background_images/browser/view_counter_model.cc

```diff
@@ -16,6 +17,23 @@ ViewCounterModel::ViewCounterModel() {
 
 ViewCounterModel::~ViewCounterModel() = default;
 
+void ViewCounterModel::SetCampaignsTotalBrandedImageCount(
+    const std::vector<size_t>& campaigns_total_image_count) {
+  campaigns_total_branded_image_count_ = campaigns_total_image_count;
+  total_campaign_count_ = campaigns_total_branded_image_count_.size();
+  for (size_t i = 0; i < total_campaign_count_; ++i) {
+    campaigns_current_branded_image_index_.push_back(0);
+  }
+  // Pick first campaign randonly.
```

Comment:
Typo

Reply:
Updated to `Pick the first campaign randomly.`